awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -590,6 +601,7 @@ const FormikApp = withFormik({
       job_type: template.job_type || 'run',
       inventory: template.inventory || '',
       project: template.project || '',
+      scm_branch: template.scm_branch || '',
```

Comment:
I'm not sure if the api will complain if you pass branch (even if it's empty string) if allow_override is false for the project.  need to pull down and test that

Reply:
@jlmitch5  It does complain, but before I submit that info I check to see if the project.allow_override:true. If it is not, then I remove that scm_branch value.

Reply:
nice